src/main/java/net/poweredbyhate/wildtp/WildTP.java

```diff
@@ -50,73 +41,24 @@
     public static boolean wb;
     public static boolean notPaper;
     public static Location cash;
-    public static HashSet<String> nonoBlocks;
+    String[] bluredLines;
+    HashMap<String, String> aliaz;
+    HashMap<String, WorldConfig> thugz;
+    HashMap<UUID, BukkitRunnable> ohWait;
 
     public void onEnable() {
-        try
-        {
-            if (Integer.valueOf(Bukkit.getBukkitVersion().split("\\.")[1]) <= 12)
-            {
-                new BukkitRunnable()
-                {
-                    @Override
-                    public void run()
-                    {
-                        getLogger().severe("This version of Wilderness-TP does not support your ancient server version.");
-                        getLogger().warning("Either update your server to 1.13, or use Wild 1.52");
-                        getLogger().warning("http://r.robomwm.com/oldwild");
-                    }
-                }.runTaskTimer(instace, 1L, 6000L);
-                return;
-            }
+        if (Integer.valueOf(Bukkit.getBukkitVersion().split("\\.")[1]) <= 12) {
+            getLogger().severe("This version of Wilderness-TP does not support your ancient server version.");
+            getLogger().warning("Either update your server to 1.13, or use Wild 1.52");
+            getLogger().warning("https://robomwm.com"); // URL changed because the old one no more works

```

Comment:
for me http://r.robomwm.com/oldwild still redirects to https://dev.bukkit.org/projects/wildernesstp-the-one-true-wildtp/files?filter-status=1&filter-game-version=2020709689%3A6588

Reply:
I'm sure having checked and double checked, this wasn't working a few hours ago... No it's ok. It's a kinda magic! (or temporary down) I've pushed a revert to the previous URL, thanks for the report.